package.json

```
@@ -72,7 +72,7 @@
     "vrtest": "cd apps && cd vr-tests && yarn screener"
   },
   "devDependencies": {
-    "@angular-devkit/schematics": "12.0.2",
+    "@angular-devkit/schematics": "=11.x.x",
```

please use exact versions when defining single version policy dev dependencies

```suggestion
    "@angular-devkit/schematics": "11.2.14",
```